Functions/CatalogItems/Remove-RsCatalogItem.ps1

```diff
@@ -25,6 +25,7 @@ function Remove-RsCatalogItem
 
     [cmdletbinding()]
     param(
+        [Parameter(mandatory,ValueFromPipeline = $true,ValueFromPipelinebyPropertyname = $true)]
```

Comment:
I don't think this parameter should be mandatory, if you pass a proxy you don't pass the server url